README.md

```diff
@@ -27,12 +27,20 @@ Here's the basic configuration you'll usually need:
   "app_name": "Test",
   "icon_path_darwin": "path/to/icon.icns",
   "icon_path_linux": "path/to/icon.png",
-  "icon_path_windows": "path/to/icon.ico"
+  "icon_path_windows": "path/to/icon.ico",
+  "version_electron": "4.0.1",
+  "version_astilectron": "0.33.0",
 }
 ```
 
 It will process the project located in the current directory and bundle it in the `output` dir for your os/arch.
 
+## Bundle Customizations
```

Comment:
I'd rather rename the chapter to `Bundle for specific Astilectron and/or Electron versions` and remove the `app_name` line

Reply:
updated README as per conversation